bench/bench.go

```diff
@@ -0,0 +1,281 @@
+package main
```

Comment:
Could we move this to `cmd/bench` (with the other binaries, is [is standard](https://go.dev/doc/modules/layout#packages-and-commands-in-the-same-repository)). The Ollama CLI sits in there but I think we can figure out how to put that into `./cmd/ollama` at some point